python/cudf/cudf/_libxx/issorted.pyx

```diff
@@ -0,0 +1,42 @@
+from cudf._libxx.cudf cimport *
+from cudf._libxx.cudf import *
+from cudf._libxx.utils cimport *
+from cudf._libxx.utils import *
+
+from cudf._libxx.includes.issorted cimport is_sorted
+
+def issorted(Table source_table, order=[], null_prec=[]):
+    """
+    Checks whether the rows of a `table` are sorted in lexicographical order.
+
+    Parameters
+    ----------
+    source_table : table whose columns are to be
+                   checked for sort order
+    order : expected sort order of each column
+            size must be len(columns) or empty
+            if empty, all columns expected sort
+            order is set to ascending
+    null_prec : desired order of null
+                compared to other elements for each column
+                size must be len(columns) or empty
+                if empty, null order is set to before
```

Comment:
Fix docstring, parameters first line should define type / allowed values / default value, second line+ describe parameter